### DISCUSSION TITLE
Solution scrollBehavior with sticky navigation header?

### DISCUSSION THREAD

**Discussion by ray1618** [cypress-io/cypress#26846] [Questions and Help]
Hi there,

My navigation header became sticky and now some test are failing because when I cy.get('').click() an element it will fail because the click will auto scroll the element to top. BUT the sticky header will be in front of it 😑.

Ofcourse there are workarounds like manualy scroll to a correct position, force click or make the viewport large enough so it doesn't need to scroll. But is there any (neat) way to offset the default `scrollBehavior: 'top'` ?

**Comment by CHBaker** [comment]
I have been wondering the same thing. you can set an offset in the scroll function, but I would like a global scroll offset so I can keep cypress auto scroll on
